cdhweb/pages/management/commands/exodus.py

```diff
@@ -238,6 +285,32 @@ def form_pages(self):
         'root': Collection.objects.get(pk=get_root_collection_id())
     }
 
+    def profile_pages(self):
+        """Exodize all existing Profiles to new ProfilePage model."""
+        # all the fields on Profile that moved to Person will have been handled
+        # by django migrations; we just need to create new Page models
+        people_landing = PeopleLandingPage.objects.first()
+        if not people_landing:
+            return
+        for profile in Profile.objects.filter(user__person__isnull=False):
+            # NOTE not sure why Profile.user.person fails; works in console?
```

Comment:
still an issue?

Reply:
i think so, yeah. probably can remove the comment since it works as-is, I just didn't understand why `Profile.user.person` didn't work